aula1/atividade/calcMulti.php

```diff
@@ -0,0 +1,70 @@
+<?php
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+$valor1 = (float) $_POST['primeiroNumero'];
+$valor2 = (float) $_POST['segundoNumero'];
+
+$resultadoFinal = '';
+function multiplicacao($valor1, $valor2)
+{
+    return $valor1 * $valor2;
+}
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
```

Comment:
olhando melhor, você pode olhar se a variavel $_POST não está vazia, se não estiver vazia é sinal que submeteu o formulário. em vez de pegar a variavel server e fazer esse if

```suggestion
if ([] !== $_POST) {
```

```suggestion
if (count($_POST) > 1) {
```